Libplanet.RocksDBStore/RocksDBStore.cs

```diff
@@ -788,9 +849,25 @@ private byte[] TxNonceKey(Address address)
                 .ToArray();
         }
 
-        private ColumnFamilyHandle GetColumnFamilyFromChainId(Guid chainId)
+        private IEnumerable<Iterator> IterateDb(RocksDb db, byte[] prefix, Guid? chainId = null)
         {
+            ColumnFamilyHandle cf = GetColumnFamily(chainId);
+            Iterator it = db.NewIterator(cf);
+            for (it.Seek(prefix); it.Valid() && it.Key().StartsWith(prefix); it.Next())
```

Comment:
It's just a question. Now it checks prefix to match one more time because of bloom filter. Then does it guarantee values not matching the prefix come in the last? If not, it can't guarantee to iterate all values matching the prefix. doesn't it?

Reply:
@moreal +1. I believe his question has a valid point.  If the result iterator can emit either matched keys and mismatched keys and they are mixed between each other (e.g., `a1`, `b1`, `a2`, `b2`, … where prefix = `a`) the current code could omit every matched key appeared later than the first unmatched key.

Reply:
Yes, it is guaranteed that mismatched keys come after the matched key because the returned key is ordered byte-wise.

Reply:
Fair enough.